uefi/src/lib.rs

```diff
@@ -51,24 +57,25 @@
 //! - `logger`: Logging implementation for the standard [`log`] crate
 //!   that prints output to the UEFI console. No buffering is done; this
 //!   is not a high-performance logger.
+//! - `panic_handler`: Add a default panic handler that logs to `stdout`.
 //! - `panic-on-logger-errors` (enabled by default): Panic if a text
 //!   output error occurs in the logger.
 //! - `unstable`: Enable functionality that depends on [unstable
 //!   features] in the nightly compiler.
 //!   As example, in conjunction with the `alloc`-feature, this gate allows
 //!   the `allocator_api` on certain functions.
+//! - `qemu`: Use the special `qemu-exit` device when a panic occurs to signal
+//!    an error. Also needs the `panic_handler` feature to make any sense.
```

Comment:
nit: I think the second sentence can be dropped. As a user I think this would sound like they need to enable `panic_handler` in addition to `qemu`, but that happens automatically.